source/hic_hal/nxp/lpc55xx/LPC55S26/gcc/startup_LPC55S26.S

```diff
@@ -0,0 +1,884 @@
+/* --------------------------------------------------------------------------*/
+/* @file:    startup_LPC55S26.s                                                */
+/* @purpose: CMSIS Cortex-M33 Core Device Startup File                       */
+/*           LPC55S26                                                          */
+/* @version: 1.1                                                      */
+/* @date:    2019-5-16                                                         */
+/* --------------------------------------------------------------------------*/
+/*                                                                           */
+/* Copyright 1997-2016 Freescale Semiconductor, Inc.                         */
+/* Copyright 2016-2021 NXP                                                   */
+/* All rights reserved.                                                      */
+/*                                                                           */
+/* SPDX-License-Identifier: BSD-3-Clause                                     */
+/*****************************************************************************/
+/* Version: GCC for ARM Embedded Processors                                  */
+/*****************************************************************************/
+
+
+    .syntax unified
+    .arch armv8-m.main
+
+    .section .isr_vector, "a"
+    .align 2
+    .globl __Vectors
+__Vectors:
+    .long   __StackTop                                      /* Top of Stack */
+    .long   Reset_Handler                                   /* Reset Handler */
+    .long   NMI_Handler                                     /* NMI Handler*/
+    .long   HardFault_Handler                               /* Hard Fault Handler*/
+    .long   MemManage_Handler                               /* MPU Fault Handler*/
+    .long   BusFault_Handler                                /* Bus Fault Handler*/
+    .long   UsageFault_Handler                              /* Usage Fault Handler*/
+    .long   SecureFault_Handler                             /* Secure Fault Handler*/
+    .long   0                                               /* Reserved*/
```

Comment:
This line and the next 2 needs to be modified include DAPLINK fields (see https://github.com/ARMmbed/DAPLink/blob/main/source/hic_hal/nxp/lpc55xx/gcc/startup_LPC55S69_cm33_core0.S#L35).

`__Vectors` should also be renamed to `__isr_vector` to be consistent with `lpc55xx.ld`